packages/trip-form/src/test-utils/modes.js

```diff
@@ -128,12 +128,10 @@ const commonModes = {
   accessModes: [
     {
       mode: "WALK",
-      showWheelchairSetting: true,
```

Comment:
These deletions of the wheelchair setting will affect demos involving accessibility. Is that intended?

Reply:
You are right we can probably bring wheelchairs back for some modes!

Reply:
It's in the car modes which still allows the setting to be tested.